stats/Export.md

```diff
@@ -0,0 +1,34 @@
+# Export API Overview
+The stats library aggregates measurements into Views (which are essentially a collection of
+metrics, each with a different set of labels). All output objects are immutable and read-only.
```

Comment:
Is not "immutable and read-only" redundant?

Reply:
Done.